apps/wallet-dashboard/components/VirtualList.tsx

```diff
@@ -54,7 +53,7 @@ export function VirtualList<T>({
             return;
         }
 
-        // Fetch the next page if the last rendered item is the one we added as extra, and there is still more pages to fetch
+        // Fetch the next page if the last item is rendered and there are still more pages to fetch
         if (lastItem.index >= items.length - 1 && hasNextPage && !isFetchingNextPage) {
```

Comment:
Looks to me like we removed part of the extra item logic (from old L38) and the "one we added as extra" explanation in the comment, but the condition assuming this logic is still in place `lastItem.index >= items.length - 1`. I don't fully understand the reasoning behind this "one extra" logic but if we don't need it we should cleanup all.